maths/area.py

```diff
@@ -1,7 +1,7 @@
 """
 Find the area of various geometric shapes
 """

```

Comment:
Add a credible source as a reference.

Reply:
Thank you @ZeroDayOwl  for you valuable suggestion.